Update image component

hstandaert opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description
<!--- Describe your changes in detail -->
* Removed figure component and merged it into `Image`
* Update image placeholder according to design (https://zpl.io/VxBpDpx)
* Added broken image placeholder

## Screenshots (if appropriate):
![screencapture-localhost-3000-components-preview-image-2019-08-29-08_10_11](https://user-images.githubusercontent.com/4415097/63914782-79146280-ca34-11e9-83ba-e2de20b2b010.png)


## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [x] I have updated the styleguide CHANGELOG accordingly.

hstandaert committed:
Add styling for broken image placeholder

hstandaert committed:
Remove figure component

hstandaert committed:
Update CHANGELOG

delrueba reviewed (changes requested):
What about the figure documentation?

hstandaert committed:
Add figure documentation to image component

hstandaert commented:
> What about the figure documentation?

Good point! I added it to the `Image` component. @mattiasberlamont, could you review whether this documentation is sufficient for the Image component?

mattiasberlamont commented on components/21-atoms/image/README.md:
```diff
@@ -19,5 +17,7 @@ For images, the following requirements should be followed:
 The image component is used in the:
 
+* <a href="{{path './teaser.html'}}">Teaser component</a>
+* <a href="{{path './teaser.html'}}">Teaser component</a>
```
```suggestion
* <a href="{{path './header.html'}}">Header component</a>
```

mattiasberlamont committed:
Updated Image documentation.

mattiasberlamont commented:
@hstandaert I could not find how to do suggestions in a proper way, so I updated the documentation myself and added a commit.

Another suggestion based on the screenshot (but I cannot change it myself):

* I would rename the variation "placeholder" to "no image placeholder".
* I would rename the variation "broken" to "broken image placeholder".

This to make clear which placeholder should be used when.

I refer to these placeholders in the updated Image documentation.

hstandaert committed:
Rename variants

hstandaert commented:
> @hstandaert I could not find how to do suggestions in a proper way, so I updated the documentation myself and added a commit.
> 
> Another suggestion based on the screenshot (but I cannot change it myself):
> 
> * I would rename the variation "placeholder" to "no image placeholder".
> * I would rename the variation "broken" to "broken image placeholder".
> 
> This to make clear which placeholder should be used when.
> 
> I refer to these placeholders in the updated Image documentation.

@mattiasberlamont, thanks for you contribution. I have updated the naming of the variants as suggested 🙂
